libdnf/swdb/item_comps_environment.cpp

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright (C) 2017 Red Hat, Inc.
+ *
+ * Licensed under the GNU Lesser General Public License Version 2.1
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#include "item_comps_environment.hpp"
+
+CompsEnvironmentItem::CompsEnvironmentItem(std::shared_ptr<SQLite3> conn)
+  : Item(conn)
+{
+}
+
+CompsEnvironmentItem::CompsEnvironmentItem(std::shared_ptr<SQLite3> conn, int64_t pk)
```

Comment:
What does `pk` mean?

Reply:
A primary key, I guess that `id` would be more obvious.